db/migrate/20200119195227_add_visited_path_to_pageviews.rb

```diff
@@ -0,0 +1,7 @@
+class AddVisitedPathToPageviews < ActiveRecord::Migration[5.2]
+  def change
+    add_column :page_views, :visited_page_path,     :string
+    add_column :page_views, :visited_page_full_url, :string
+    add_column :page_views, :visited_page_category, :string
```

Comment:
Assuming we are going to do some lookups with these we might want some indexes, at the very least I would suspect we want one on category.

Reply:
Yeah, I avoided this at first because I didn't want to do it prematurely, but that is almost certainly going to be a thing.

Reply:
+1 for adding indexes only when needed